tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -55,15 +55,15 @@ spec:
 
         if [ ! -f "${RPA_FILE}" ] ; then
             echo "No ReleasePlanAdmission file was found."
-            echo "false" | tee $(results.mapped.path)
```

Comment:
I assume the issue was the new line character, so `echo -n` probably would've fixed it too, but I am fine with printf